docs/tutorials/tutorial-sketches-theta.md

```diff
@@ -209,36 +145,22 @@ Let's first see what the data looks like in Druid. Run the following SQL stateme
 SELECT * FROM ts_tutorial
 ```
 
-![View data with SELECT all query](../assets/tutorial-theta-06.png)
+![View data with SELECT all query](../assets/tutorial-theta-03.png)
```

Comment:
nit/not necessary for this pr: In general, I think tables represent query results better than screen shots. And you can also wrap the results in a `view results` expander/details section to make the overall document smaller.